subsys/bluetooth/mesh/lpn.c

```diff
@@ -150,11 +153,6 @@ static void friend_clear_sent(int err, void *user_data)
 {
 	struct bt_mesh_lpn *lpn = &bt_mesh.lpn;
 
-	/* We're switching away from Low Power behavior, so permanently
-	 * enable scanning.
-	 */
-	bt_mesh_scan_enable();
-
```

Comment:
Could you move this to a separate commit, please? Also, clarify where the scanning does get enabled if it doesn't happen here.

Reply:
scanning will be enable after `FRIEND_REQ_RETRY_TIMEOUT` seconds.